### PR TITLE
Update rummager integration

### PR DESCRIPTION
**Pull request by dwhenry**
This moves to using the new V2 API which provides synchronous updates to the ES index behind rummager.

This fixes the following issues:

- [x]  I removed the Greece travel advice page from the Kos query. No effect.  (looking now, I’m guessing this was because it was the list item on the query)
- [ ] I deleted https://www.gov.uk/government/publications/protecting-children-from-radicalisation-the-prevent-duty from the query IS. The top best bet, https://www.gov.uk/government/news/isil-uk-government-response then disappeared as well (without me deleting it). That query still shows up in search at top spot. But if you look in search admin at the IS query, it appears to have no links within it.
- [ ] The search results preview window hasn’t worked since we unlocked indexing a few weeks back

https://trello.com/c/K5zkvpfi/537-delete-best-bet-query-and-items-not-working-in-search-admin

### CODE REVIEW

**Comment by dwhenry** [comment]
Could not duplicate point 2 so will leave open.

**Comment by dwhenry** [comment]
POint 4 is most likely caused by finder-frontend not allow embedding from different domains and should be fixed separately in that app.

File: Gemfile [hunk 1]
```diff
@@ -14,11 +14,7 @@ gem 'gds-sso', '~> 13.5.0'
 gem 'govuk_admin_template'
 gem "govuk_app_config", "~> 0.2.0"
 
-if ENV["API_DEV"]
```
Comment (brenetic) [inline Gemfile]: Why no more dev option?

File: features/support/rummager.rb [hunk 2]
```diff
@@ -1,6 +1,29 @@
-Before do
+# Before do
```
Comment (brenetic) [inline features/support/rummager.rb]: Should these still be commented out?

File: Gemfile (hunk 1 above)
Comment (dwhenry) [reply Gemfile]: It is now considered bad practise and has been removed from most of the other app (as far as I am aware)

File: features/support/rummager.rb (hunk 2 above)
Comment (dwhenry) [reply features/support/rummager.rb]: they should be deleted, done now

**Pull request merged**